pkgs/servers/openafs/1.8/module.nix

```diff
@@ -37,63 +37,7 @@ stdenv.mkDerivation {
 
   buildInputs = [ libkrb5 ];
 
-  patches = [
-    # Import of code from autoconf-archive
-    (fetchpatch {
-      url = "https://git.openafs.org/?p=openafs.git;a=patch;h=d8205bbb482554812fbe66afa3c337d991a247b6";
-      hash = "sha256-ohkjSux+S3+6slh6uZIw5UJXlvhy9UUDpDlP0YFRwmw=";
-    })
-    # Use autoconf-archive m4 from src/external
-    (fetchBase64Patch {
-      url = "https://gerrit.openafs.org/changes/14944/revisions/ea2a0e128d71802f61b8da2e44de3c6325c5f328/patch";
-      hash = "sha256-PAUk/MXL5p8xwhn40/UGmo3UIhvl1PB2FwgqhmqsjJ4=";
-    })
-    # cf: Use common macro to test compiler flags
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
It is still referenced by `fetchBase64Patch`, which I’d like to keep around because it’s almost guaranteed to be needed for future kernel upgrades.

Reply:
missed that, sorry